src/content/drafts/redesigning-eslint.md

```diff
@@ -0,0 +1,106 @@
+
+![An example of the Space Grotesk font](/assets/images/blog/2022/space-grotesk.png)
+
+Space Grotesk works hand-in-hand with it’s origin sister font, Space Mono. It was developed for editorial use in headline and display typography, the letterforms infuse a geometric foundation and grotesque details with qualities often found in headline typefaces of the 1960s, many of which have since been co-opted by science fiction films, television, and literature.
+
+![An example of the Space Mono font](/assets/images/blog/2022/spacemono-satellites.gif)
+
+### The Colours
+
+ESLint’s colour palette speaks to the brand in ways that are every bit as powerful as the copy and logo. It not only affect how the brand design looks, but can go as far as to elicit emotion and reflect the personality of the ESLint brand. Colors also play an important role in how our designs "look & feel", and are the first impression.
+
+While color can be subjective, it's clear that good colors can attract and convert users. Bad colors, on the other hand, can turn away users and even make designs inaccessible and unusable.
+
+Due to the functional nature of ESLint, we chose to keep it simple with a primary, neutral and small set of secondary palettes. The new primary colour palette is derived from the original logo — we use the two existing colours to create a unique primary tonal range.
+
+![A snapshot of ESLint's color palette](/assets/images/blog/2022/eslint-colors.png)
+
+## The Website
+
+Next up was the redesign of the [ESLint website](https://eslint.org/). Besides maximising the visual appeal of the site, we had a few goals we wanted to achieve for each page. For example, on the homepage:
+
+1. We wanted to include the latest and upcoming versions of ESLint in the hero section. This was a key part of the design, as it would be the first time we had a dedicated section for ESLint updates.
+2. We wanted to promote sponsorship. ESLint relies on donations for ongoing maintenance and development, and we wanted to make sure that we could help accelerate the growth of the project.
+3. We wanted to show ESLint in action — how it can be used in the real world, and how it can be used to make your code more readable.
+4. We wanted to showcase some of the incredible stats that ESLint has amassed — as of right now, that's 9M dependants, 25.7M weekly downloads and 21K stars on GitHub.
+
+![A mockup of the new ESLint homepage design](/assets/images/blog/2022/eslint-website-home.png)
+
+I also wanted to introduce some animation and life into the ESLint homepage hero while giving our audience a sense of what it's like to work with ESLint.
+
+The red underline in VS Code is synonymous with ESLint, as is the way VS Code implements ESLint action items. Using these visual indicators, I created a small but fun interaction that gives our visitors the simplified experience.
```

Comment:
Could we add in the animated gif of the animation to this section?

Reply:
Yeah definitely - I was hoping to do that! Is there a way of using video (mp4, stream from Vimeo maybe) or does it need to be gif?